moveit_ros/perception/CMakeLists.txt

```diff
@@ -46,20 +46,23 @@ find_package(catkin REQUIRED COMPONENTS
 
 find_package(Eigen3 REQUIRED)
 find_package(OpenMP REQUIRED)
-find_package(OpenCV)
+find_package(octomap REQUIRED)
+find_package(OpenCV REQUIRED)
```

Comment:
Does any of the functionality either here or in the related PRs rely on specific version of OpenCV (or have some minimum version requirement?) OpenCV API has changed somewhat from major version 2 to 3 to 4 after all.

Reply:
Basically, the ARUCO target detection is working against OpenCV 3.x and 4.x. But according to my test, the performance on 3.2 is very odd and really bad. Unfortunately 3.2 is the default version comes with Ubuntu 18.04. But OpenCV compilation time is too long to put into CI, so I think I would add a minimum version here and leave some comments in tutorial to indicate a better version. Thanks for mention this!